### ISSUE TITLE
Broken link to ASCII chart in 'char' reference page

### ISSUE THREAD

**Issue by arnopluk** [arduino/reference-en#882]
In the reference page about the char datatype (https://www.arduino.cc/reference/en/language/variables/data-types/char/) there is a broken link to the ASCII chart (https://www.arduino.cc/en/Reference/ASCIIchart).

**Comment by kengdahl** [comment]
@arnopluk Is this [the one](https://docs.arduino.cc/built-in-examples/communication/ASCIITable) that was supposed to be linked to?

**Comment by per1234** [comment]
@kengdahl no. You can see what the page contained in the archived copy from the Internet Archive Wayback Machine:

https://web.archive.org/web/20171028220458/https://www.arduino.cc/en/Reference/ASCIIchart
